### PR TITLE
Update acs-javascript-calling-library-release-notes.md

### PR DESCRIPTION
**Pull request by ankeshni**

### CODE REVIEW

File: releasenotes/acs-javascript-calling-library-release-notes.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ If you are working with **Teams users**, please follow the `Teams identities` do
...
 - [Teams Identities object model](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-with-voice-video-calling-custom-teams-client#azure-communication-services-calling-web-sdk-object-model)
 
+## 1.22.1-beta.1 (2024-02-03 TODO)
+- Available in NPM - TODO [https://www.npmjs.com/package/@azure/communication-calling/v/1.19.2-beta.2](https://www.npmjs.com/package/@azure/communication-calling/v/1.19.2-beta.2)
+
+### Fixes
+ - Improved the PPTLive feature so that it only shows the active signal after the call has been connected.
+ - Always render N number of videos in a large meeting. Dominant speakers video will be rendered as priority then non-dominant speakers by their join order (if need to have N videos).
+ - Enabled auto-acceptance for transfer requests within the transfer feature. Replaced the `transferRequested` event with `transferAccepted` to facilitate automatic acceptance.
+
```
Comment (enricohuang) [inline releasenotes/acs-javascript-calling-library-release-notes.md:20]: Fixes:
- Fix the video source when starting a video call with raw video stream

Comment (cn0151) [reply releasenotes/acs-javascript-calling-library-release-notes.md:20]: Could you replace Replaced the `transferRequested` event with `transferAccepted` to facilitate automatic acceptance. with Replaced the `transferRequested` event with `transferAccepted` due to auto-transfer acceptance

**Review by cn0151 — COMMENTED** [review]
Could you replace Replaced the transferRequested event with transferAccepted to facilitate automatic acceptance. with Replaced the transferRequested event with transferAccepted due to auto-transfer acceptance

**Comment by ankeshni** [comment]
> Could you replace Replaced the transferRequested event with transferAccepted to facilitate automatic acceptance. with Replaced the transferRequested event with transferAccepted due to auto-transfer acceptance.

Updated

File: releasenotes/acs-javascript-calling-library-release-notes.md [hunk 2]
```diff
@@ -10,6 +10,16 @@ If you are working with **Teams users**, please follow the `Teams identities` do
 - [Teams Identities quick start](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-with-voice-video-calling-custom-teams-client)
 - [Teams Identities object model](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-with-voice-video-calling-custom-teams-client#azure-communication-services-calling-web-sdk-object-model)
 
+## 1.22.1-beta.1 (2024-02-03 TODO)
+- Available in NPM - TODO [https://www.npmjs.com/package/@azure/communication-calling/v/1.19.2-beta.2](https://www.npmjs.com/package/@azure/communication-calling/v/1.19.2-beta.2)
+
```
Comment (enricohuang) [inline releasenotes/acs-javascript-calling-library-release-notes.md:15]: Features:
- Support one incoming video with Full HD(1080p) resolution

File: releasenotes/acs-javascript-calling-library-release-notes.md [hunk 3]
```diff
@@ -10,6 +10,27 @@ If you are working with **Teams users**, please follow the `Teams identities` do
...
+ - Improved selectMicrophone/selectSpeaker input validation.
+
+## 1.21.1 stable (2024-02-07)
+- Available in NPM - TODO [https://www.npmjs.com/package/@azure/communication-calling/v/1.21.1](https://www.npmjs.com/package/@azure/communication-calling/v/1.21.1)
+
+### New Features:
+- Support receiving one incoming video stream with Full HD(1080p) resolution
+
+### Fixes
+- Improved selectMicrophone/selectSpeaker input validation.
```
[the remark this answers is not in the document]
Comment (ankeshni) [reply releasenotes/acs-javascript-calling-library-release-notes.md]: updated

**Pull request merged**